matrix/common/constants.py

```diff
@@ -504,10 +511,14 @@ class MetadataSchemaName(Enum):
 fields, <code>genes_detected</code> for example, are calculated during secondary analysis.
 Full descriptions of those fields are forthcoming.</p>
 
-<h3><code>genes.tsv.gz</code></h3>
+<h3><code>features.tsv.gz</code></h3>
 <p>The gene metadata contains basic information about the genes in the count matrix.
 Each row is a gene, and each row corresponds to the same row in the expression mtx file.
 Note that <code>featurename</code> is not unique.</p>
+
+<h3><code>barcodes.tsv.gz</code></h3>
+<p>A list of cell barcodes corresponding to the columns found in matrix.mtx.gz.
+Note that barcodes may not be unique.</p>
```

Comment:
@mckinsel What do you recommend we use instead to ensure uniqueness across projects? Will storing something other than barcodes in this file cause confusion?